app/components/learnergroup-cohort-user-manager.js

```diff
@@ -49,13 +58,11 @@ export default class LearnergroupCohortUserManagerComponent extends Component {
 
   @action
   toggleUserSelectionAllOrNone() {
-    const selectedUsers = this.selectedUsers.length;
-    const filteredUsers = this.filteredUsers.length;
-
-    if (selectedUsers >= filteredUsers) {
-      this.selectedUsers = [];
+    const unselectedFilteredUsers = this.filteredUsers.filter(user => { return ! this.selectedUsers.includes(user); });
```

Comment:
As a one liner this should be `const unselectedFilteredUsers = this.filteredUsers.filter(user => !this.selectedUsers.includes(user));`
Otherwise it should be split as:
```
const unselectedFilteredUsers = this.filteredUsers.filter(user => { 
  return ! this.selectedUsers.includes(user); 
});
```

Reply:
handled.